src/main/java/seedu/address/logic/commands/SearchCommand.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.recipe.Recipe;
+
+/**
+ * Displays all recipes that uses the ingredient within the recipe book
+ */
+public class SearchCommand extends Command {
+    public static final String COMMAND_WORD = "search";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays all recipes that uses the ingredient within the recipe book "
+            + "Parameters: "
+            + "ingredient name "
+            + "Example: " + COMMAND_WORD + " flour";
+
+    public static final String MESSAGE_SUCCESS = "%$s ingredients listed";
```

Comment:
Don't know if this needs to be here

Reply:
May i know the reason why you think it need not be there

Reply:
I think it does need to be there though